enable/tests/primitives/test_image.py

```diff
@@ -110,15 +113,17 @@ def test_image_gc_32(self):
         image_gc = self.image_32._image
         assert_array_equal(image_gc.bmp_array, self.data)
 
+    @unittest.skipIf(six.PY3, reason="Crashes on python 3. See GH #95.")
     def test_draw_24(self):
-        gc = GraphicsContext((256, 128), pix_format='rgba32')
+        gc = GraphicsContext((256, 128), pix_format='rgb24')
```

Comment:
I think the original test code wasn't wrong. It's testing drawing a 24bit image to a 32-bit destination.

We weren't testing drawing to a 24bit destination before, so your changes don't need to be reverted.